docs/7-dashboard-ui/3-Zarf-deployment-ui.md

```diff
@@ -0,0 +1,114 @@
+# Zarf Deployment Web UI
+
+Zarf has a Deployment Web UI built in that supports a number of Zarf features used during the package deployment process. For users who prefer not to use the command line tool, the Web UI creates a simple experience to deploy and manage Zarf clusters and packages. The Web UI can be used to connect to existing clusters (via a Kubeconfig), initialize a cluster, deploy packages into a cluster, update packages in the cluster, and remove packages from the cluster. 
+
+The Zarf Web UI mirrors the functionality of the Zarf CLI commands, but with a more intuitive flow and familiar web application patterns for non-technical users. The web UI does not offer any additional commands or core functionality to Zarf. 
+
+## Open the Zarf Deployment Web UI
+
+The Zarf Deployment Web UI can easily be spun up with a single command from the CLI. 
+
+Follow these steps to get started using the Web UI
+
+1. Step one: [Install the Zarf binary](/docs/3-getting-started.md)
+2. Step two: Open a terminal shell
+3. Step three: Type in the following command: ```zarf dev UI```
+
+![GIF showing the Web UI lauch from the CLI terminal](../.images/dashboard/Web_UI__Launch_w__Cluster_AdobeExpress.gif)
+
+## Using the Zarf Deployment Web UI
+
+### Cluster Connection Status
+
+When Zarf is running it automatically searches for a Kubeconfig on the local machine. If the Kubeconfig is found, it searches the default cluster to determine if it is a Zarf cluster (i.e. initialized). There are two different cluster statuses the Web UI will display based on the state of the cluster found. 
+
+#### Cluster not Connected (Not Initizalized)
+
+![Web UI shows organge warning status and message "cluster not connected" on the cluster card](../.images/dashboard/Web%20UI%20-%20Cluster%20Not%20Connected.png)
+
+1. Shown when there is no Kubeconfig found on the machine.
+2. Shown when a Kubeconfig is found on the machine, but Zarf has not been deployed and set up in the cluster. 
+
+#### Cluster Connected (Initialized)
+
+If Zarf finds a cluster in the Kubeconfig that has Zarf resources in it it will automatically connect to the cluster and display the cluster details on the Web UI.
+
+![Web UI shows cluster metat data in on the cluster card when a connected cluster is found](../.images/dashboard/Web%20UI%20-%20Status%20Cluster%20connected.png)
+
+1. Shown when there is a Kubeconfig found on the machine with a default cluster that has Zarf resources in it.
+
+
+### Connect to Existing Cluster 
+
+The Zarf Web UI makes connecting to existing clusters easy. When on the packages page, if there is no Zarf cluster currently connected, select the connect cluster button. If Zarf finds a Kubeconfig it will ask the user if they want to connect to the the default cluster context. 
```

Comment:
```suggestion
The Zarf Web UI makes connecting to existing clusters easy. When on the packages page, if there is no Zarf cluster currently connected, select the connect cluster button. If Zarf finds a Kubeconfig it will ask the user if they want to connect to the default cluster context. 
```